### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by christianonoh**
# ToDoList

## ✨ _Implemented Main Features_ ✨

- Set up this project with webpack from previous exercise. `(Requested)`
- Created an empty To Do List placeholder (`ul` element). `(Requested)`
- The index.html file is set as the template using the HTML Webpack Plugin. `(Requested)`
- Created an array of objects for the tasks. `(Requested)`
- The list appears in order of the index values for each task. `(Requested)`

## ✨ _Other Features_  ✨
-  _Made sure there were no linter errors._  `(Requested)`
-  _Used the correct GitHub Flow._ `(Requested)`
-  _Followed best practices for HTML, CSS & JavaScript_ `(Requested)`

# 🎈 _Thank you for your review!_ 🎈

### CODE REVIEW

**Review by Omar-Muhamad — COMMENTED** [review]
# Invalid ❌

Hi @christianonoh,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] You didn't document your work professionally:
   - You forgot to edit your README file. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and edit it with your personal and project data.
   
- [ ] Some linters checks are not passed. Please make sure that all checks are passing and green.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by Omar-Muhamad — COMMENTED** [review]
# Invalid ❌

Hi @christianonoh,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] You didn't document your work professionally:
   - You forgot to edit your README file. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and edit it with your personal and project data.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,276 @@
...
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```
Comment (Shedrack-Sunday) [inline README.md:26]: - [ ] Kindly delete this commented part of your codebase to keep the public repo clean. Its not a good practice to have such in your project as it serves as a guide 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,276 @@
...
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LICENSE -->
```
Comment (Shedrack-Sunday) [inline README.md:268]: - [ ] Please check y detailed review on line `3` regarding this section 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,276 @@
...
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) licensed.
+
+<!-- _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._ -->
```
Comment (Shedrack-Sunday) [inline README.md:274]: - [ ] Here also, delete this suggested part-out, to keep your code clean and professional 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @christianonoh        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights:
✔️ good code structure
✔️ good PR summary
✔️NICE UI
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- A few best practices adjustments are needed👏

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @christianonoh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**